docs/adr/index.md

```diff
@@ -22,6 +22,7 @@ the decision records that have been made.
 - [0008 - Decision Points are Ordered Sets](0008-decision-points-are-ordered-sets.md)
 - [0009 - Outcomes are Ordered Sets](0009-outcomes-are-ordered-sets.md)
 - [0010 - Outcome Sets are separate from Decision Point Groups](0010-outcome-sets-are-separate-from-decision-point-groups.md)
+- [0011 - Correspondence between Automatable v2.0.0, Value Density v1.0.0, and CVSS v4](0011-automatable-and-value-density-and-CVSSv4.md)
```

Comment:
Need to add/commit/push the file that is the target of this link.

Reply:
Well that's embarrassing. Anyway, fixed.